draft-ietf-scitt-architecture.md

```diff
@@ -492,25 +492,23 @@ Might dereference to:
 }
 ```
 
-### Naming Artifacts
+### Support for multiple Artifacts
 
-Many Issuers issue Signed Statements about different Artifacts under the same DID, so it is important for everyone to be able to immediately recognize by looking at the Envelope of a Signed Statements what Artifact it is referring to.
+Many Issuers produce Signed Statements about various Artifacts under the same Identity. Issuers and Verifiers must be able to recognize the Artifact to which the statements pertain by looking at the Signed Statement.
 This information is stored in the Feed header of the Envelope.
 Issuers MAY use different signing keys (identified by `kid` in the resolved key manifest) for different Artifacts, or sign all Signed Statements under the same key.
 
-### Signed Statement Metadata
+### Registration Policy Metadata
 
-Besides Issuer, Feed and kid, the only other mandatory metadata in a Signed Statement is the type of the Payload, indicated in the `cty` (content type) Envelope header.
-However, this set of mandatory metadata is not sufficient to express many important Registration Policies.
-For example, a Registry may only allow a Signed Statement to be registered, if it was signed recently.
-While the Issuer is free to add any information in the payload of the Signed Statements, the Transparency Services (and most of its Auditors) can only be expected to interpret information in the Envelope.
+SCITT payloads are opaque to Transparency Services, so Registration Policy decisions can only be based on interpretation of information in the Envelope.
 
-Such metadata, meant to be interpreted by the Transparency Services during Registration Policy evaluation, SHOULD be added to the `reg_info` header, unless the data is private (in which case, it MAY be sent to the Transparency Service as an additional input during registration).
+The small mandatory set of metadata in the envelope of a Signed Statement is neither intended nor sufficient to express the information required for the processing of Registration Policies in a Transparency Service.
```

Comment:
Why should this statement be part of the specification text? Maybe we should try to fix the meaning of Reg_Info?

Please find a first attempt to bash at #113

Reply:
#113 was merged, and helps. This appears to add clarity and doesn't seem to conflict.

I'm also thinking Registration Policy Metadata is the right heading, unless we rename Reg_Info

Reply:
This is a great topic as we dig deeper into receipts. Please open a new issue for receipts, or we can have that conversation in #103